pyemma/coordinates/__init__.py

```diff
@@ -53,6 +34,27 @@
    cluster_uniform_time
    assign_to_centers
 
+Classes
+=======
+**Coordinate classes** encapsulating complex functionality. You don't need to construct these classes yourself, as this
+is done by the user API functions above. Find here a documentation how to extract features from them.
+
+.. autosummary::
+   :toctree: generated/
+
+   pipelines.Pipeline
+   transform.PCA
+   transform.TICA
 
 """
 from .api import *
+
+from .pipelines import Pipeline
```

Comment:
BTW: if possible please merge today, becase we'll be using the docs 
tomorrow.

Am 19/04/15 um 23:59 schrieb Martin K. Scherer:
>
> In pyemma/coordinates/__init__.py 
> <https://github.com/markovmodel/PyEMMA/pull/265#discussion_r28657404>:
>
> >
> >  """
> >  from .api import *
> > +
> > +from .pipelines import Pipeline
>
> First you wanted to hide those, so we underscore imported them in api 
> and now you lift them here? Makes no sense to me.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/markovmodel/PyEMMA/pull/265/files#r28657404>.
>


-- 

----------------------------------------------
Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de
Mail:  Arnimallee 6, 14195 Berlin, Germany
----------------------------------------------